src/CMakeLists.txt

```diff
@@ -122,44 +122,37 @@ set(GISH_H
     config.h
 )
 
-if(WIN32)
-    list(APPEND GISH_H video/glext.h)
-endif(WIN32)
-
 if(APPLE)
     list(APPEND GISH_SRCS sdl/SDLMain.m)
     list(APPEND GISH_H sdl/SDLMain.h)
 endif(APPLE)
 
-if(WIN32)
-    add_executable(freegish WIN32 ${GISH_SRCS} ${GISH_H})
-else()
-    add_executable(freegish ${GISH_SRCS} ${GISH_H})
-endif()
+add_executable(freegish ${GISH_SRCS} ${GISH_H})
+
+target_include_directories(freegish PRIVATE
+    ${SDL2_INCLUDE_DIRS}
+    ${OPENAL_INCLUDE_DIR}
+    ${OPENGL_INCLUDE_DIR}
+    ${Vorbis_INCLUDE_DIR}
+    ${PNG_INCLUDE_DIR}
+)
+
+target_link_libraries(freegish
+    ${SDL2_LIBRARIES}
+    ${OPENAL_LIBRARY}
+    ${OPENGL_LIBRARIES}
+    ${Vorbis_LIBRARIES}
+    ${PNG_LIBRARIES}
+)
+
+if(LIBM)
+  target_link_libraries(freegish m)
+endif(LIBM)
+
+set_property(TARGET freegish PROPERTY VS_DEBUGGER_WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}")
```

Comment:
I did not know this existed. What does it do?

Reply:
I didn't try this to be honest, but when generating a vsproj file (for older VS versions), it would set this as the working directory for the debugger (when pressing play). It isn't compatible with VS2022 yet apparently.

I've removed it again, as the binary is now in root of project again.